module-apps/application-settings/ApplicationSettings.cpp

```diff
@@ -692,6 +692,18 @@ namespace app
                            ::settings::SettingsScope::Global);
     }
 
+    auto ApplicationSettings::getWallpaperOption() -> gui::WallpaperOption
+    {
+        return static_cast<gui::WallpaperOption>(utils::getNumericValue<int>(
+            settings->getValue(::settings::Wallpaper::option, ::settings::SettingsScope::Global)));
```

Comment:
Well accordingly to settings guidelines it should be Local and probably handled by APM in that case (I do not agree with that -> for me it is ok).